internal/engine/eval/vulncheck/vulncheck.go

```diff
@@ -134,6 +144,34 @@ func (e *Evaluator) Eval(ctx context.Context, pol map[string]any, res *engif.Res
 	return nil
 }
 
+// getPatchedVersion returns a version that patches all known vulnerabilities. If no such version exists, it returns
+// the version that patches the most vulnerabilities. If none of the vulnerabilities have patches, it returns the
+// empty string.
+func getPatchedVersion(vulns []Vulnerability) (fixedVersion string, latest bool, noFix bool) {
```

Comment:
Did you consider adding the three return parameters to the `Vulnerability` structure as unexported (lowercase) attributes and then calling `getPatchedVersion` directly from within the queryVulnDb function?

Reply:
True, it can probably go in `VulnerabilityResponse`. I'll create a followup issue.